### DISCUSSION TITLE
视频能不能支持一下ts文件

### DISCUSSION THREAD

**Discussion by MoChen-github** [AlistGo/alist#2049] [Invalid question]
视频能不能支持一下ts文件

**Comment by yuyoiuyoiu** [comment]
直接在管理→设置→预览→视频类型里加上ts就行

**Reply by liueic** [reply]
我试了一下 显示无法播放

**Reply by pongfcnkl** [reply]
@liusomes 
自己手动用ffmpeg转码 参考h.264 aac mp4

**Comment by liueic** [comment]
> @liusomes 自己手动用ffmpeg转码 参考h.264 aac mp4
我刚才解决了 本地用ffmpeg转码然后再传到云盘 现在云盘里面还有很多这种ts文件 我想能不能在线转换或者把云盘挂载在本地然后再用ffmepg

**Comment by kaixinxiaowangzi666** [comment]
把在alist里把视频后缀名改成mp4就可以播放了
